src/main/java/core/basesyntax/Diamond.java

```diff
@@ -6,13 +6,32 @@
 
     /**
      * Нарисовать ромб со звёздочек заданной величины.
-     *
      * Пример: величина 3, значит максимальное кол-во звёзд в ряду (по диагонали) будет 3.
-     *
      * Вместо System.out.print() или System.out.println() использовать stream.print()
      * или stream.println()
      */
     public static void drawDiamond(int maxSize, PrintStream stream) {
+        for (int i = 1; i <= maxSize; i++) {
+            for (int j =  maxSize - i; j > 0; j--) {
+                stream.print(" ");
+            }
+            for (int k = 0; k < i - 1; k++) {
+                stream.print("*");
```

Comment:
```suggestion
                stream.print("* ");
```
and then we can remove next line